spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/ReadIntegrationTestBase.java

```diff
@@ -364,4 +364,43 @@ public void testArrowCompressionCodec() {
 
     assertThat(avroResults).isEqualTo(arrowResultsForLZ4FrameCodec);
   }
+
+  @Test
+  public void testStringExpressions() {
+    Dataset<Row> df = spark.read().format("bigquery").load(TestConstants.SHAKESPEARE_TABLE);
+    df =
+        df.selectExpr(
+            "word",
+            "ASCII(word) as ascii",
+            "LENGTH(word) as length",
+            "LOWER(word) as lower",
+            "LPAD(word, 10, '*') as lpad",
+            "RPAD(word, 10, '*') as rpad",
+            "TRANSLATE(word, 'a', '*') as translate",
+            "TRIM(concat('    ', word, '    ')) as trim",
+            "LTRIM(concat('    ', word, '    ')) as ltrim",
+            "RTRIM(concat('    ', word, '    ')) as rtrim",
+            "UPPER(word) as upper",
+            "INSTR(word, 'a') as instr",
+            "INITCAP(word) as initcap",
+            "CONCAT(word, '*', '!!') as concat");
+    df = df.limit(2);
```

Comment:
Please add .where("word = 'augurs'") as order is not guaranteed

Reply:
Added this